nemo/collections/asr/models/ctc_models.py

```diff
@@ -691,9 +691,15 @@ def _setup_transcribe_dataloader(self, config: Dict) -> 'torch.utils.data.DataLo
         Returns:
             A pytorch DataLoader for the given audio file(s).
         """
-        batch_size = min(config['batch_size'], len(config['paths2audio_files']))
+        if 'manifest_filepath' in config:
```

Comment:
What about RNNT overrides ? Add the support for it's two methods, even if it's not used yet. All transcribe related info should ideally be synchronized among all model implementations.

Reply:
good point. I totally forgot rnnt. :(

Reply:
added support for rnnt and rnnt bpe

Reply:
Cool.